lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -106,6 +110,16 @@ def self.guess(config)
           return {}
         end
 
+        def self.valid_json?(json_object)
+          if json_object == "null"
+              return false
+          end
+          JSON.parse(json_object)
```

Comment:
I think put the statement into begin end make code easier to read
`begin
  JSON.parse(string)
   return true
rescue JSON::ParserError => e  
  # do smth
end `

Reply:
Done in https://github.com/treasure-data/embulk-input-google_analytics/pull/39/commits/6f1986ee7156c34fd497099e468e218eb6a38c2a